etc/languages.tsv

```diff
@@ -1,80 +1,80 @@
-ID	Doculect	SubGroup	ISO639P3code	Language_ID	Glottocode	Longitude	Latitude	Sources
+ID	Language	Family	ISO639P3code	Language_ID	Glottocode	Longitude	Latitude	Source
```

Comment:
Please modify "Language" in the google sheet to "Name", and then re-submit hte languages tsv file. I think this will allow you to run the code with cldf bench.

Reply:
Thank you for the explanation, I am sure it is some minor detail I missed somewhere. I did not try changing anything in the ID column (and only looked for possible Ntogapid_Itogapuk with the underscore). Right now I changed "language" to "name" in the google sheet file and pusehd it. I will also attach the error message I get to this comment.
[error-message-ntogapid-itogapuk.txt](https://github.com/tupian-language-resources/tuled/files/6159227/error-message-ntogapid-itogapuk.txt)